File: php/status.php
<?php

$apiKey = "YOUR_API_KEY";
$endpoint = "https://app.pendless.com/api/enqueue/";

$data = [
    "procedure" => "status",
    "uuid" => "TASK_UUID_HERE"
];

$options = [
    "http" => [
        "header"  => [
            "Authorization: Bearer $apiKey"
        ],
        "method"  => "POST",
        "content" => json_encode($data)
    ]
];

$context = stream_context_create($options);
$result = file_get_contents($endpoint, false, $context);

$response = json_decode($result, true);
print_r($response);

File: csharp/Status.cs
using System;
using System.Net.Http;
using System.Text;
using System.Threading.Tasks;

class StatusExample
{
    static async Task Main()
    {
        string apiKey = "YOUR_API_KEY";
        string endpoint = "https://app.pendless.com/api/enqueue/";

        var data = new
        {
            procedure = "status",
            uuid = "TASK_UUID_HERE"
        };

        using (var client = new HttpClient())
        {
            client.DefaultRequestHeaders.Add("Authorization", $"Bearer {apiKey}");
            var content = new StringContent(System.Text.Json.JsonSerializer.Serialize(data), Encoding.UTF8, "application/json");

            var response = await client.PostAsync(endpoint, content);
            string respBody = await response.Content.ReadAsStringAsync();
            Console.WriteLine(respBody);
        }
    }
}
